css/20_map.css

```
@@ -185,6 +185,17 @@ g.vertex.selected .shadow {
     fill-opacity: 0.7;
 }
 
+/* circle to visualize radius/diameter */
+g:is(.point, .vertex) .radius {
+    display: none;
+}
+g:is(.point, .vertex):is(.active, .selected) .radius {
+    display: block;
```

This made me wonder what `display` value is the correct one to use for SVG elements. 😅 Couldn't find anything authoritative at a first glance, but Firefox' and Chrome's dev tools say that it's `inline` if not set (which is AFAIK simple the default value of `display` in CSS). OTOH, the [`display` SVG attribute](https://developer.mozilla.org/en-US/docs/Web/SVG/Reference/Attribute/display) only really defines what `none` should do. So, maybe here it makes more sense to just unset the previously set `display: none`??


```suggestion
    display: unset;
```

//PS: setting the `display:none` is probably not even needed, see below